crypto/evp/pmeth_lib.c

```diff
@@ -355,6 +359,12 @@ int EVP_PKEY_CTX_ctrl(EVP_PKEY_CTX *ctx, int keytype, int optype,
 
 }
 
+int EVP_PKEY_CTX_ctrl_uint64(EVP_PKEY_CTX *ctx, int keytype, int optype,
+                                int cmd, uint64_t value)
```

Comment:
Formatting nit. There is corresponding misalignment in include/openssl/evp.h. It wouldn't be inappropriate to fix it in new request, @johndoe31415.